sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_common/_base_rai_svc_eval.py

```diff
@@ -17,32 +17,28 @@ class RaiServiceEvaluatorBase(EvaluatorBase):
     This includes content safety evaluators, protected material evaluators, and others. These evaluators
     are all assumed to be of the "query and response or conversation" input variety.
 
-    param eval_metric: The evaluation metric to be used for evaluation. This is used by the API call logic
-    to specify which evaluation to perform.
-    type eval_metric: ~azure.ai.evaluation._common.constants.EvaluationMetrics
-    param eval_last_turn: If True, only the last turn of the conversation will be evaluated, and no
+    :param eval_metric: The evaluation metric to be used for evaluation. This is used by the API call logic
+        to specify which evaluation to perform.
+    :type eval_metric: ~azure.ai.evaluation._common.constants.EvaluationMetrics
+    :param eval_last_turn: If True, only the last turn of the conversation will be evaluated, and no
         aggregation will be performed. If False, all turns will be evaluated and the numeric results will be,
         aggregated. Per-turn results are still be available in the output via the "evaluation_per_turn" key
         when this occurs. Default is False, resulting full conversation evaluation and aggregation.
-    type eval_last_turn: bool
+    :type eval_last_turn: bool
     """
 
     @override
     def __init__(
         self,
         eval_metric: EvaluationMetrics,
         azure_ai_project: dict,
-        credential: Optional[dict] = None,
```

Comment:
Just recall something. We should remove below code in rai_service.py

![image](https://github.com/user-attachments/assets/81ff6a49-dab1-45e6-a9ec-f32bd330604c)

Reply:
good call, I had this change locally but looks like I didn't push it